superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -218,6 +234,10 @@ class Header extends React.PureComponent {
         interval,
         chartCount: affectedCharts.length,
       });
+      this.props.addWarningToast(
+        t(`This dashboard will force refresh at every ${intervalMessage}`),
```

Comment:
You should provide args for formatting this message instead of using string templates as per this api: https://github.com/apache-superset/superset-ui/tree/master/packages/superset-ui-translation#api

Reply:
fixed